spec/features/transactions_filter_spec.rb

```diff
@@ -404,5 +405,20 @@
         end
       end
     end
+
+    context 'should display without transfers amounts' do
+      let!(:transaction) { create :transaction, :income, bank_account: ba, organization: org }
+      let(:amount)       { money_with_symbol Money.new(transaction.amount, org.default_currency) }
+      let!(:transfer)    { create :transfer, :with_different_currencies, bank_account_id: ba.id }
+
+      before do
+        visit root_path
+        within "##{dom_id(transaction)}" do
+          click_link ba.to_s
+        end
+      end
+
+      it { expect(page).to have_content("Total: #{amount}") }
```

Comment:
@yunixon you should check this content not in whole page. You need to check that it showed in specific cell of totals table.